vars/piperPipelineStageSecurity.groovy

```diff
@@ -57,6 +61,20 @@ void call(Map parameters = [:]) {
             }
         }
 
+        if (config.detectExecuteScan) {
+            securityScanMap['Detect'] = {
+                node(config.nodeLabel) {
+                    try{
+                        durationMeasure(script: script, measurementName: 'detect_duration') {
+                            detectExecuteScan script: script
+                        }
+                    }finally{
+                        deleteDir()
```

Comment:
not sure why this is needed, but I see checkmarx is doing it too. is it cargo cult, or do we know we need it for some reason?

Reply:
If this is omitted you would leave a dirty workspace which can cause issues for the next user of the workspace.